manual-deployment.md

```diff
@@ -171,7 +171,7 @@ For more information, see [Configure Replication Zones](configure-replication-zo
 You can run the built-in SQL client from any machine with the `cockroach` binary, CA cert, client certificate, and client key. Make sure the machine you want to use has these files and then start the client:  
 
 ~~~ shell
-$ ./cockroach sql --url=postgresql://root@<node1-hostname>:26257/?sslcert=root.cert&sslkey=root.key&sslmode=verify-full&sslrootcert=ca.cert
+$ cockroach sql --url=postgresql://root@<node1-hostname>:26257/?sslcert=root.cert&sslkey=root.key&sslmode=verify-full&sslrootcert=ca.cert
```

Comment:
just noticed this now: the url needs to be quoted, or the shell will barf on `&`.